.gitattributes

```
@@ -129,8 +129,9 @@ knapsack_rspec_report.json      lockfile
 package-lock.json               lockfile
 pnpm-lock.yaml                  lockfile
 
-### Force text diff for Gemfile.lock
+### Force text diff for specific lockfiles
 Gemfile.lock                    diff
+yarn.lock                       diff
```

```suggestion
yarn.lock                       diff
knapsack_rspec_report.json      diff
```

I ran into this when updating the knapsack report too, could we include it as well?

---

Turns out that file was explicitly listed in the stanza above, I tried to clean it up and with 2081b367c46e4ad85140941117b00320742ebb1b